cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,51 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
+  const user = {
+    username: 'v66',
+    password: 'Pa7$7777'
+  }
 
-  });
+  const book = {
+    title: 'Speaking JavaScript',
+    description: 'Like it or not, JavaScript is everywhere'
+  }
 
-  it('', () => {
+  before(() => {
+    cy.visit('/login');
+  });
 
+  it('should provide the ability to login, search, add and delete book', () => {
```

Comment:
You have to create a custom login command for login via API and then use it when it is needed

Reply:
Done